resources/thaumadditions/blockstates/amber_lamp.json

```diff
@@ -0,0 +1,11 @@
+{
+    "forge_marker": 1,
+    "defaults": {
+        "model": "minecraft:layered_cube_all_2",
+        "textures" : { "top_all" : "thaumadditions:blocks/amber_lamp_glow", "bot_all" : "thaumadditions:blocks/amber_lamp" },
+        "transform": "forge:default-block"
+    },
+    "variants": {
+        "normal": [{}], "inventory": [{"model": "minecraft:layered_cube_all_2", "textures": {"top_all" : "thaumadditions:blocks/amber_lamp", "bot_all" : "thaumadditions:blocks/amber_lamp" }}]
```

Comment:
Using `layered_cube_all_2` with same textures `top_all` and `bot_all` makes no sense - it makes two same cube textures over each other. In this case you may use cube with one texture instead.
Also, Amber Lamp doesnt need two states, since its static block, always glowing. It have same behavior as, for example, `Lit Redstone Lamp`.